README.md

```diff
@@ -18,14 +20,31 @@ Predict next word
 ```python
 # test model input: وَاللَّهُ
 model.next_word(input())
-# output [('عَلِيمٌ', 24), ('لَا', 19), ('يَعْلَمُ', 16)]
+# output: [('عَلِيمٌ', 24), ('لَا', 19), ('يَعْلَمُ', 16)]
 
 # test model input: وَاللَّهُ بِمَا تَعْمَلُونَ
 model.next_word(input())
-# output [('خَبِيرٌ', 7), ('بَصِيرٌ', 6), ('عَلِيمٌ', 2)]
+# output: [('خَبِيرٌ', 7), ('بَصِيرٌ', 6), ('عَلِيمٌ', 2)]
 ```
 
 Pre-trained model is provided in ``lm_alquran.pickle`` and training corpus is provided in ``alquran_korpus.txt``. Corpus is obtained from [Al Quran Cloud](http://api.alquran.cloud/) using simple arabic edition.
 
+## Sambung Ayat
+Use this to find similar option as ground truth answer and use it as options. Use either ``TFIDF`` from ``lm.tfidf`` or ``BM25`` from ``lm.bm25``
+```python
+from lm.tfidf import TFIDF
+
+# Load and train
+tfidf = TFIDF()
+tfidf.fit(ayahs)
+```
+
+Predict next word
+```python
+# Get top 3 similar ayahs
+tfidf.get_top_n(ayahs[10], ayahs, n=3, print_top=True)
+# output: [(654, 24.528617979554483), (552, 20.823947791102057), (727, 16.009677537122926)]
```

Comment:
The example output is still using the `scores[id]` instead of `corpus[id]`, like in the `lm/tfidf.py` and `example_usage.ipynb`.